maven-projects/info/pom.xml

```diff
@@ -42,6 +42,9 @@
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <os.plugin.version>1.6.2</os.plugin.version>
+        <protobuf.plugin.version>0.6.1</protobuf.plugin.version>
+        <protobuf.version>3.21.1</protobuf.version>
```

Comment:
I would suggest to pin the version not only for Maven builds but globally for the whole GraphAr. Otherwise we may face problem one day if we will try to serialize from java and, for example, read from cpp

Reply:
Yes, we can follow our [doc](https://github.com/apache/incubator-graphar/blob/main/format/README.md) and pin it at 1.32.0

Reply:
1.32.0 is the version of buf, not `protoc`.

Th actual version of protobuf that we are using is `3.27.1`
![image](https://github.com/apache/incubator-graphar/assets/29755009/70e5744c-7033-4be6-b37a-538a8b1ef3e4)

(explanation: we are on `proto3` syntax and the version of protobuf plugin for a minor version of `protoc` itself)

I mean, I have a strong feeling, that we need to pin it globally. Otherwise it may become hard to maintain it in two different places (Maven and buf.gen.yaml)

Reply:
> 1.32.0 is the version of buf, not `protoc`.
> 
> Th actual version of protobuf that we are using is `3.27.1` ![image](https://private-user-images.githubusercontent.com/29755009/346524005-70e5744c-7033-4be6-b37a-538a8b1ef3e4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0MzgxNjMsIm5iZiI6MTcyMDQzNzg2MywicGF0aCI6Ii8yOTc1NTAwOS8zNDY1MjQwMDUtNzBlNTc0NGMtNzAzMy00YmU2LWIzN2EtNTM4YThiMWVmM2U0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA4VDExMjQyM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM5YmQ3YjEzMTI4NTEwZDhlOWI4MjU4OWJkYWIzODZiNDRlYWE4ZmU0ZGEzNWJhOWY3YWEzYWY2Yjc3OGIxYzgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qel9nGrZEPteyrPRLA_vcZiPOphCJpq5VB-V-QSS0qM)
> 
> (explanation: we are on `proto3` syntax and the version of protobuf plugin for a minor version of `protoc` itself)
> 
> I mean, I have a strong feeling, that we need to pin it globally. Otherwise it may become hard to maintain it in two different places (Maven and buf.gen.yaml)

agree

Reply:
The current protoc version is **3.27.1**; at least we may pin it in Maven properties

If we are not going to use buf for generatign Java code, we do not need to touch buf version from this PR